packages/block-editor/src/components/global-styles/filters-panel.js

```diff
@@ -179,48 +225,36 @@ export default function FiltersPanel( {
 					isShownByDefault={ defaultControls.duotone }
 					panelId={ panelId }
 				>
-					<Dropdown
-						popoverProps={ popoverProps }
-						className="block-editor-global-styles-filters-panel__dropdown"
-						renderToggle={ ( { onToggle, isOpen } ) => {
-							const toggleProps = {
-								onClick: onToggle,
-								className: clsx( { 'is-open': isOpen } ),
-								'aria-expanded': isOpen,
-							};
-
-							return (
-								<ItemGroup isBordered isSeparated>
-									<Item as="button" { ...toggleProps }>
-										<LabeledColorIndicator
-											indicator={ duotone }
-											label={ __( 'Duotone' ) }
+					<ItemGroup isBordered isSeparated>
```

Comment:
According to [the README](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/item-group/item-group/README.md), the ItemGroup component must be used with an Item component.

> ItemGroup should be used in combination with the [Item sub-component](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/item-group/item/README.md).

Can we just remove the `ItemGroup` component? Ideally, there would be no need for an `ItemGroup` component since the filter panel only has one Duotone control.